server/src/main/java/org/elasticsearch/inference/InferenceServiceRegistry.java

```diff
@@ -20,7 +20,7 @@
 import java.util.function.Function;
 import java.util.stream.Collectors;
 
-public class InferenceServiceRegistry extends AbstractLifecycleComponent {
```

Comment:
I don't believe we need this to be an `AbstractLifecycleComponent` anymore since it's not part of the `NodeConstruction` class right?